smoke-tests/apps/JMS/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/JmsControllerSpansEnabledTest.java

```diff
@@ -1,131 +0,0 @@
-/*
- * ApplicationInsights-Java
- * Copyright (c) Microsoft Corporation
- * All rights reserved.
- *
- * MIT License
- * Permission is hereby granted, free of charge, to any person obtaining a copy of this
- * software and associated documentation files (the ""Software""), to deal in the Software
- * without restriction, including without limitation the rights to use, copy, modify, merge,
- * publish, distribute, sublicense, and/or sell copies of the Software, and to permit
- * persons to whom the Software is furnished to do so, subject to the following conditions:
- * The above copyright notice and this permission notice shall be included in all copies or
- * substantial portions of the Software.
- * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
- * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
- * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
- * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
- * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
- * DEALINGS IN THE SOFTWARE.
- */
-
-package com.microsoft.applicationinsights.smoketest;
-
-import static com.microsoft.applicationinsights.smoketest.WarEnvironmentValue.JAVA_8;
-import static org.assertj.core.api.Assertions.assertThat;
-
-import com.microsoft.applicationinsights.smoketest.schemav2.Data;
-import com.microsoft.applicationinsights.smoketest.schemav2.Envelope;
-import com.microsoft.applicationinsights.smoketest.schemav2.RemoteDependencyData;
-import com.microsoft.applicationinsights.smoketest.schemav2.RequestData;
-import java.util.List;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.RegisterExtension;
-
-@Environment(JAVA_8)
-@UseAgent("controller_spans_enabled_applicationinsights.json")
-class JmsControllerSpansEnabledTest {
```

Comment:
did you mean to delete this?

Reply:
no, i merged main a couple of times.. it's back now.

Reply:
how do i look up which commit it belongs to so that i can have an idea why it was deleted?

Reply:
try https://stackoverflow.com/questions/7203515/how-to-find-a-deleted-file-in-the-project-commit-history

Reply:
look liked it was deleted in [this commit](https://github.com/microsoft/ApplicationInsights-Java/pull/2439/commits/bd12ff8eb89b3e61ec481afde8b9f7a51cbd5f7f). must be a mistake.